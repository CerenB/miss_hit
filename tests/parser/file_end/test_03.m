% (c) Copyright 2019 Florian Schanda

function test_03()
   disp potato
end   
