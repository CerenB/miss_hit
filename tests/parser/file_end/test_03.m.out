In test_03.m, line 4
|    disp potato
|         ^^^^^^ style: end statement with a semicolon
----------------------------------------------------------------------
--  Parse tree for test_03.m
Root: Function_File
  Functions: Function_Definition
    Signature: Function_Signature
      Name: Identifier <test_03>
    Body: Sequence_Of_Statements
      Statements: Naked_Expression_Statement
        Expression: Command form Function_Call
          Name: Identifier <disp>
          Arguments: Char_Array_Literal 'potato'
----------------------------------------------------------------------
In test_03.m, line 3
| function test_03()
|                 ^ info: this token is not linked to the ast
In test_03.m, line 3
| function test_03()
|                  ^ info: this token is not linked to the ast
In test_03.m, line 5
| end   
| ^^^ info: this token is not linked to the ast
MISS_HIT Summary: 1 file(s) analysed, 1 style issue(s)
